packages/cli/src/zostso/issue/command/Command.definition.ts

```diff
@@ -34,10 +34,24 @@ export const CommandDefinition: ICommandDefinition = {
     ],
     options: ([
         {
+            // Old API behavior will be utilized upon specifying --ssm to be false,
+            // otherwise try new API and if it fails, fallback to old API.
+
+            // Specifying --ssm to be false makes the value of --stateful have no impact on
```

Comment:
I think that there is a relationship between the 'suppress-startup-messages' and 'stateful' options within the zosmf APIs that I do not fully understand. None-the-less, from the perspective of someone using the command, it does not seem like 'suppress-startup-messages' should control 'stateful'. It seems like we should think about it as 'stateful' will cause us to override 'suppress-startup-messages' to the desired value.

I am very late to the review of this PR, so this choice may have already been fully considered, and it could be reasonable to not change any of the paradigm at this late time.